server.js

```diff
@@ -1,27 +1,66 @@
-import express from "express";
-import cors from "cors";
-
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// import avocadoSalesData from "./data/avocado-sales.json";
-// import booksData from "./data/books.json";
-// import goldenGlobesData from "./data/golden-globes.json";
-// import netflixData from "./data/netflix-titles.json";
-// import topMusicData from "./data/top-music.json";
-
-// Defines the port the app will run on. Defaults to 8080, but can be overridden
-// when starting the server. Example command to overwrite PORT env variable value:
-// PORT=9000 npm start
+import express from "express"; 
+import cors from "cors"; 
+import goldenGlobesData from "./data/golden-globes.json";
+
 const port = process.env.PORT || 8080;
 const app = express();
 
-// Add middlewares to enable cors and json body parsing
 app.use(cors());
-app.use(express.json());
+app.use(express.json()); 
 
-// Start defining your routes here
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  res.send({responseMessage: "Golden Globe stats, library at /globes"}); 
+});
+
+ app.get("/globes/:ceremony", (req, res) => {
+  const ceremonyNumber = goldenGlobesData.filter((globe) => {
+    return globe.ceremony === +req.params.ceremony;
+  });
+  if (ceremonyNumber) {
+      res.status(200).json({
+      success: true,
+      message: "OK",
+      response: {
+      goldenGlobesData: ceremonyNumber
+    }
+  });
+} else {
+  res.status(404).json(`Ceremony number ${ceremonyNumber} not found`); //This part does not work 
```

Comment:
This was a really nice thought! Interesting approach to try to implement what the user typed... 
I think that you are missing that the json should be an object. You can try with this: 

```
res.status(404).json({
      success: false,
      message: "Ceremony not found",
      response: {}
    });
```

And then in the message part, try with `${ceremony}` ? or `${:ceremony}` or `${params.ceremony}` and see if it works.